nautobot_secrets_providers/providers/__init__.py

```diff
@@ -3,13 +3,13 @@
 from .aws import AWSSecretsManagerSecretsProvider, AWSSystemsManagerParameterStore
 from .azure import AzureKeyVaultSecretsProvider
 from .hashicorp import HashiCorpVaultSecretsProvider
-from .delinea import ThycoticSecretServerSecretsProviderId, ThycoticSecretServerSecretsProviderPath
+from .delinea import DelineaSecretServerSecretsProviderId, DelineaSecretServerSecretsProviderPath
 
 __all__ = (  # type: ignore
     AWSSecretsManagerSecretsProvider,  # pylint: disable=invalid-all-object
     AWSSystemsManagerParameterStore,  # pylint: disable=invalid-all-object
     AzureKeyVaultSecretsProvider,  # pylint: disable=invalid-all-object
+    DelineaSecretServerSecretsProviderId,  # pylint: disable=invalid-all-object
+    DelineaSecretServerSecretsProviderPath,  # pylint: disable=invalid-all-object
     HashiCorpVaultSecretsProvider,  # pylint: disable=invalid-all-object
-    ThycoticSecretServerSecretsProviderId,  # pylint: disable=invalid-all-object
-    ThycoticSecretServerSecretsProviderPath,  # pylint: disable=invalid-all-object
 )
```

Comment:
Let's fix this so that pylint can stop complaining :-). Each of these should be a string rather than the actual class object, e.g.:

```py
__all__ = (
    "AWSSecretsManagerSecretsProvider",
    "AWSSystemsManagerParameterStore",
 ....
```

Reply:
Doing this leads to an `AttributeError` from `secrets.py`:
```
nautobot-1  |     if provider.is_available:
nautobot-1  |        ^^^^^^^^^^^^^^^^^^^^^
nautobot-1  | AttributeError: 'str' object has no attribute 'is_available'
```

so I needed to also update `secrets.py`:
```
for provider_name in providers.__all__:
    provider = getattr(providers, provider_name) <<<<<<<<<<<<<<<<<<
    # Don't publish multiple times.
    if provider in secrets_providers:
        continue

    if provider.is_available:
        secrets_providers.append(provider)
```
which is good now.